tests/conf/oscd.yaml

```diff
@@ -0,0 +1,17 @@
+model:
+  class_path: ChangeDetectionTask
+  init_args:
+    loss: 'ce'
+    model: 'unet'
+    backbone: 'resnet18'
+    in_channels: 13
```

Comment:
If stacking the channels should this be 2 * 13?

Reply:
OK and it is not necessary to config T?

Reply:
`in_channels` is multiplied by 2 in `configure_models` when initializing Unet.